modules/storage-page-memory/src/test/java/org/apache/ignite/internal/storage/pagememory/configuration/PageMemoryDataRegionValidatorImplTest.java

```diff
@@ -46,51 +50,120 @@
 @ExtendWith(ConfigurationExtension.class)
 public class PageMemoryDataRegionValidatorImplTest {
     @InjectConfiguration(polymorphicExtensions = UnsafeMemoryAllocatorConfigurationSchema.class)
-    private PageMemoryStorageEngineConfiguration engineConfig;
+    private VolatilePageMemoryStorageEngineConfiguration volatileEngineConfig;
+
+    @InjectConfiguration(polymorphicExtensions = UnsafeMemoryAllocatorConfigurationSchema.class)
+    private PersistentPageMemoryStorageEngineConfiguration persistentEngineConfig;
 
     @Test
     void testValidationFail() {
```

Comment:
I would suggest to split this test into 3, it's quite difficult to read otherwise

Reply:
Fix it